src/modules/store/store.ts

```diff
@@ -0,0 +1,3 @@
+import { writable } from 'svelte/store';
+
+export const user = writable({ uid: '', name: '', allowed: false, isLoggedIn: false });
```

Comment:
せっかく ts で書くので、 ユーザーオブジェクトの中身の type を定義した方が良いかなと
少し調べたところ
```ta
import { writable, Writable } from 'svelte/store';
```

```suggestion
type User = {
  ...
}

export const user: Writable<User> = writable({ uid: '', name: '', allowed: false, isLoggedIn: false });
```

こんな書き方ができるみたいです

https://javascript.plainenglish.io/writing-a-svelte-store-with-typescript-22fa1c901a4

Reply:
修正しました